Gemfile.lock

```diff
@@ -75,29 +75,29 @@ GEM
       rest-client (>= 1.7.3, < 3.0)
       rspec (~> 3.1)
     analytical (3.0.12)
-    annotate (2.7.3)
+    annotate (2.7.4)
```

Comment:
Yikes! That makes sense. I'm not sure how to only update dependencies that are needed for one gem. This is my first commit since launching Coyote locally, so they happened at some point when I was bundling. Do you have advice on how to do that or where I can learn more? @subelsky

Reply:
@subelsky Ah... Just read your comment below which answers my question above. So basically, next time I should bundle and if I see a ton of dependancies are updated, go ahead an commit those. Then add all the `similar_text` code and commit seperately?

Reply:
It's probably because after you added the gem you ran `bundle update` instead of `bundle install` , because `similar_text` doesn't seem to have any dependencies (https://rubygems.org/gems/similar_text/versions/0.0.4). Not worth changing now, unless it affects the build passing (one of the upgraded gems could have included a breaking change)

Reply:
...then the next commit would be for changes to the files (or additions of files) related to that dependency